src/sw.js

```diff
@@ -0,0 +1,24 @@
+self.addEventListener('push', function(event) {
```

Comment:
It will be removed so this `sw.js` needs to be renamed to `push-service.js` (or similar) and needs to be included in `sw-precache` with [importScripts()](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers#Importing_scripts_and_libraries). I'll update this PR.